domains/allancoding.json

```diff
@@ -1,11 +1,13 @@
 {
     "owner": {
         "username": "allancoding",
-        "email": "allancoding.dev@gmail.com"
+        "email": "allancoding.dev@gmail.com",
+        "twitter": "allancoding",
+        "discord": "allancoding"
     },
     "record": {
-        "A": ["217.174.245.249"],
+        "CNAME": "allancoding.github.io",
```

Comment:
Cannot have a CNAME in conjunction with other records.

Reply:
there is no A record